STEER/ESD/AliESDInputHandlerRP.cxx

```diff
@@ -99,7 +99,8 @@ Bool_t AliESDInputHandlerRP::Init(Option_t* opt)
     {
       TString rppath;
       if (det->TestBit(kReadFromArchiveBIT)) {
-	if (fPathName->EndsWith("root_archive.zip"))
+	// if (fPathName->EndsWith("root_archive.zip"))
+	if (fPathName->Contains("root_archive"))
```

Comment:
Would it work if you checked both if it contains "/root_archive" and ".zip" (with, without a regexp I don't care). Also, can you just delete the commented line?